code/modules/projectiles/projectile.dm

```diff
@@ -130,17 +130,19 @@
 			if(kill_count < 1)
 				qdel(src)
 				return
-			kill_count--
-			if((!( current ) || loc == current))
-				current = locate(Clamp(x+xo,1,world.maxx),Clamp(y+yo,1,world.maxy),z)
-			step_towards(src, current)
-			if((original && original.layer>=2.75) || ismob(original))
-				if(loc == get_turf(original))
-					if(!(original in permutated))
-						Bump(original, 1)
+			if(!paused)

```

Comment:
Wouldn't it be better to remove the bullet from the processing list and then readd it at the end of the timewarp?